src/cppcache/src/CqQueryVsdStats.cpp

```diff
@@ -108,21 +98,21 @@ CqQueryStatType::CqQueryStatType()
 ////////////////////////////////////////////////////////////////////////////////
 
 CqQueryVsdStats::CqQueryVsdStats(const char* cqqueryName) {
-  CqQueryStatType* regStatType = CqQueryStatType::getInstance();
+  auto& regStatType = CqQueryStatType::getInstance();
 
-  StatisticsType* statsType = regStatType->getStatType();
+  StatisticsType* statsType = regStatType.getStatType();
 
-  GF_D_ASSERT(statsType != NULL);
+  GF_D_ASSERT(statsType != nullptr);
```

Comment:
`static_assert` is a compile time assertion. In this case the assertion can't be resolved at compile time since the variables and functions involved are not `constexpr`, templates or other compiler resolved things.